packages/fractor/src/Configuration/FractorConfigurationBuilder.php

```diff
@@ -81,7 +82,7 @@ public function __invoke(ContainerConfigurator $containerConfigurator): void
             $parameters->set(Option::CACHE_CLASS, $this->cacheClass);
             SimpleParameterProvider::setParameter(Option::CACHE_CLASS, $this->cacheClass);
         }
-        if ((new CiDetector())->isCiDetected()) {
+        if (StaticPHPUnitEnvironment::isPHPUnitRun() || (new CiDetector())->isCiDetected()) {
```

Comment:
IMO we should not do this if a CLI param was supplied for configuring the cache, otherwise we cannot test the cache in CI or in any PHPUnit test.

Reply:
This is what Rector does. I'm open for improvements.

Reply:
Okay, let's leave it at that then for now, if we discover we want to test that separately, we can add something.